docs/babelconfigjs.md

```diff
@@ -0,0 +1,97 @@
+---
+title: babel.config.js
+id: babelconfigjs
+---
+
+## Lookup behavior
+
+The lookup behavior is direct as Babel will only look for a `babel.config.js` in the root directory (where your `package.json` is by default). If configuration is needed, see [specifying the root folder](babelconfigjs.md#specifying-the-root-folder).
```

Comment:
Maybe below we can create a file diagram to explain how it ignores the other config files

```
a/
--babel.config.js
--b/
----.babelrc // ignored
----b.js // current file
```

something like that kind of example